happy/HappyState.py

```diff
@@ -78,6 +79,7 @@ class HappyState(State):
         -i --id     Displays all known state IDs.
         -a --all    Displays the network topology state for all known states.
         -n --node   Displays all information for a node
+        -e --extension Displays happy extension value (eg. weave-fabric-id)
```

Comment:
Please align the formatting throughout the docstring

Reply:
OK, will change it to be aligned